src/api/util.js

```diff
@@ -43,29 +55,12 @@ export const blockfrostRequest = async (endpoint, headers, body) => {
       },
       method: body ? 'POST' : 'GET',
       body,
-    }).then((res) => res.json());
-    // in case blockfrost throws error 500 => loop until result in 100ms interval
-    if (result.status_code === 500) {
-      const interval = setInterval(async () => {
-        const result = await fetch(provider.api.base(network.node) + endpoint, {
-          headers: {
-            ...provider.api.key(network.id),
-            ...headers,
-            'User-Agent': 'nami-wallet',
-          },
-          method: body ? 'POST' : 'GET',
```

Comment:
The only difference between this code section and the one that does the first request is the use of the `’Cache-Control’` header. I have intentionally made it so that the `’Cache-Control’` header is always set, even in the subsequent requests, but of course I could make it so this header is only added on the first request.